js/app/src/lite/Playground.svelte

```diff
@@ -213,9 +238,12 @@
 
 	.code-header {
 		padding-top: 0.25rem;
```

Comment:
This `padding-top` should be removed?
The "app.py" text is vertically aligned to the "Run" text.
<table>
<tr>
<th>With it</th>
<th>Without it</th>
</tr>
<tr>
<td>
<img src="https://github.com/gradio-app/gradio/assets/3135397/0b265c34-bb31-4c03-a1c2-2e1da322e8f5" />
</td>
<td>
<img src="https://github.com/gradio-app/gradio/assets/3135397/a070f7c1-9df0-46af-96eb-86be33a7a1dc" />
</td>
</tr>
</table>